test/format/config.sh

```diff
@@ -139,6 +139,8 @@ CONFIG configuration_list[] = {
 
 {"cache.minimum", "minimum cache size (MB)", C_IGNORE, 0, 0, 100 * 1024}
 
+{"cache.maximum", "maximum cache size (MB)", C_IGNORE, 0, 0, 100 * 1024}
```

Comment:
I don't quite remember the way these structs are laid out - is `100 * 1024` the default? If so, if this is in MiB, that seems like a lot.

Reply:
The `100 * 1024` default value denotes maximum value that can be set for this configuration.

Reply:
Oh I see - thanks. Is that optional? If so, I see no reason to limit the maximum cache size, especially if `smoke.sh` already finds a limit and sets it.

(My personal take on the limits is that they're the values beyond which the config makes no sense or the test isn't expected to work, and the cache theoretically doesn't have an upper limit like it does a lower limit. I can definitely see us running `test/format` on huge machines and hitting the 100GB limit.)

Reply:
No for test/format the maximum value is mandatory unfortunately.

Reply:
Ok, that's unfortunate. I would make this 10TB to match the API's cache size limits, since the restriction should ideally be implemented elsewhere (like you've done in `format.sh`.

Reply:
Have configured to UINT_MAX

Reply:
Thanks - that works too.